crypto/params_dup.c

```diff
@@ -234,3 +235,12 @@ void OSSL_PARAM_free(OSSL_PARAM *params)
         OPENSSL_free(params);
     }
 }
+
+size_t ossl_param_nelem(const OSSL_PARAM *plist)
+{
+    size_t n;
+
+    for (n = 0; !ossl_param_is_empty(plist); plist++)
+        n++;
+    return n;
```

Comment:
Should this be +1 for the terminator.. (I guess you are treating this like strlen)

Reply:
Yep.